testing/utiltest.cc

```diff
@@ -1,44 +1,78 @@
-#include <gtest/gtest.h>
+#include "common.hh"
 
 #include "game/util.hh"
 
-namespace {
-	TEST(UnitTest_Utils, clamp) {
-		EXPECT_EQ(0.0, clamp(-1.0, 0.0, 1.0));
-		EXPECT_EQ(0.5, clamp(0.5, 0.0, 1.0));
-		EXPECT_EQ(1.0, clamp(2.0, 0.0, 1.0));
-	}
-
-	TEST(UnitTest_Utils, smoothstep_1) {
-		EXPECT_EQ(0.0, smoothstep(0));
-		EXPECT_EQ(0.5, smoothstep(0.5));
-		EXPECT_EQ(1.0, smoothstep(1.0));
-	}
-
-	TEST(UnitTest_Utils, smoothstep_1_clamping) {
-		EXPECT_EQ(0.0, smoothstep(-1.0));
-		EXPECT_EQ(1.0, smoothstep(2.0));
-	}
-
-	TEST(UnitTest_Utils, smoothstep_3) {
-		EXPECT_EQ(0.0, smoothstep(-1., 1., -1.0));
-		EXPECT_EQ(0.5, smoothstep(-1., 1., 0.0));
-		EXPECT_EQ(1.0, smoothstep(-1., 1., 1.0));
-	}
-
-	TEST(UnitTest_Utils, smoothstep_3_clamping) {
-		EXPECT_EQ(0.0, smoothstep(-1., 1., -2.0));
-		EXPECT_EQ(1.0, smoothstep(-1., 1., 2.0));
-	}
-
-	TEST(UnitTest_Utils, format) {
-		auto const time = std::chrono::seconds(7260);
-
-		// next two tests are not working when compiled with mingw. see https://sourceforge.net/p/mingw-w64/bugs/793/
-		//EXPECT_EQ("01/01/70 02:01", format(time, "%D %R"));
-		//EXPECT_EQ("02:01:00 1970-01-01", format(time, "%T %F"));
-		EXPECT_EQ("01/01/70 02:01", format(time, "%m/%d/%y %H:%M", true));
-		EXPECT_EQ("02:01:00 1970-01-01", format(time, "%H:%M:%S %Y-%m-%d", true));
-		EXPECT_EQ("02:01:00 1970-01-01", format(time, "%X %Y-%m-%d", true));
-	}
+TEST(UnitTest_Utils, clamp) {
+    EXPECT_EQ(0.0, clamp(-1.0, 0.0, 1.0));
+    EXPECT_EQ(0.5, clamp(0.5, 0.0, 1.0));
+    EXPECT_EQ(1.0, clamp(2.0, 0.0, 1.0));
+}
+
+TEST(UnitTest_Utils, smoothstep_1) {
+    EXPECT_EQ(0.0, smoothstep(0));
+    EXPECT_EQ(0.5, smoothstep(0.5));
+    EXPECT_EQ(1.0, smoothstep(1.0));
+}
+
+TEST(UnitTest_Utils, smoothstep_1_clamping) {
+    EXPECT_EQ(0.0, smoothstep(-1.0));
+    EXPECT_EQ(1.0, smoothstep(2.0));
+}
+
+TEST(UnitTest_Utils, smoothstep_3) {
+    EXPECT_EQ(0.0, smoothstep(-1., 1., -1.0));
+    EXPECT_EQ(0.5, smoothstep(-1., 1., 0.0));
+    EXPECT_EQ(1.0, smoothstep(-1., 1., 1.0));
+}
+
+TEST(UnitTest_Utils, smoothstep_3_clamping) {
+    EXPECT_EQ(0.0, smoothstep(-1., 1., -2.0));
+    EXPECT_EQ(1.0, smoothstep(-1., 1., 2.0));
+}
+
+TEST(UnitTest_Utils, format) {
+    auto const time = std::chrono::seconds(7260);
+
+    // next two tests are not working when compiled with mingw. see https://sourceforge.net/p/mingw-w64/bugs/793/
+    //EXPECT_EQ("01/01/70 02:01", format(time, "%D %R"));
+    //EXPECT_EQ("02:01:00 1970-01-01", format(time, "%T %F"));
```

Comment:
Maybe we should add a compiler-if-check here so they are exclued for mingw (or we delete these two entirely) 

i don't like commented code which does nothing :P